test/functional/cmdLineTests/jvmtitests/playlist.xml

```diff
@@ -785,4 +785,43 @@
 			<impl>ibm</impl>
 		</impls>
 	</test>
+	<test>
+		<testCaseName>cmdLineTester_jvmtitests_Java21andUp</testCaseName>
+		<variations>
+			<variation>Mode110</variation>
+			<variation>Mode148</variation>
+			<variation>Mode149</variation>
+			<variation>Mode610</variation>
+		</variations>
+		<command>$(ADD_JVM_LIB_DIR_TO_LIBPATH) \
+	$(JAVA_COMMAND) $(CMDLINETESTER_JVM_OPTIONS) \
+	-DTEST_ROOT=$(Q)$(TEST_RESROOT)$(Q) \
+	-DJAR=$(Q)$(TEST_RESROOT)$(D)jvmtitest.jar$(Q) \
+	-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS) -Xdump $(SQ) \
+	-DMODE_HINTS=$(Q)$(MODE_HINTS)$(Q) \
+	-jar $(CMDLINETESTER_JAR) \
+	-config $(Q)$(TEST_RESROOT)$(D)jvmtitests_Java21andUp.xml$(Q) \
+	-explainExcludes \
+	-xids all,$(PLATFORM),$(JCL_VERSION) -xlist $(Q)$(TEST_RESROOT)$(D)jvmtitests_excludes_$(JDK_VERSION).xml$(Q) -nonZeroExitWhenError; \
+	${TEST_STATUS}</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<types>
+			<type>native</type>
+		</types>
+		<versions>
+			<version>21+</version>
+		</versions>
+		<features>
+			<feature>AOT:explicit</feature>
```

Comment:
Is this AOT feature? See doc for AOT feature tag: https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#run-tests-against-specific-feature

Reply:
Thanks for pointing this out. Also, is there documentation on the meaning of different modes? Might need to update modes as well.

Reply:
For modes, you can see https://github.com/adoptium/TKG/blob/master/resources/modes.xml